docs/source/conf.py

```diff
@@ -0,0 +1,64 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'EVT-core'
+copyright = '2021, EVT'
+author = 'EVT'
+
+# The full version, including alpha/beta/rc tags
+release = '0.1.2'
```

Comment:
Could we set the version in CmakeLists.txt as an environment variable and use that here to prevent two updates being necessary?

Reply:
Ahh good catch, meant to address this. A common way I've seen is to have a file with the version so that different tools know where to look for the version. I've just updated the system (both docs and CMakeLists.txt) to read in version from a file.